website/static/js/fangorn.js

```diff
@@ -2230,6 +2259,23 @@ tbOptions = {
     uploads : true,         // Turns dropzone on/off.
     columnTitles : _fangornColumnTitles,
     resolveRows : _fangornResolveRows,
+    // Move obj.data.attributes contents to obj.data , for WB v1 API
+    lazyLoadPreprocess: function(obj) {
+        function moveAttrs(obj) {
```

Comment:
Minor: since this is an internal utility function, prefix the name with an underscore.  Also, add a docstring explaining its purpose and mentioning the `obj.attributes.attributes` issue.  Super nitpicky: rename it to `_promoteAttrs` to indicate where/how attributes is being moved.